persistence/artist_repository.go

```diff
@@ -350,15 +353,16 @@ func (r *artistRepository) Read(id string) (interface{}, error) {
 }
 
 func (r *artistRepository) ReadAll(options ...rest.QueryOptions) (interface{}, error) {
-	role := "total"
 	if len(options) > 0 {
 		if v, ok := options[0].Filters["role"].(string); ok {
-			role = v
+			role := v
```

Comment:
This change means that if there is no Role selected in the UI, it will use the max(albumartist,artist) counts. Meaning that if I have a composer with 100 songs, no albums, they would show up after an artist with just one participation. That does not feel right

Reply:
My problem is, Navidrome _only_ returns credits for album artist/artist. So, even if we show total, it isn't _really_ useful? If the artist page (and `/api/album`) supports all roles, them I'd be more comfortable with that.